README.md

```diff
@@ -20,10 +20,13 @@ rake build
 
 Prototypical contacts application that can talk LDAP.
 
-Available for iOS, Android and Windows Phone.
+Available for Android and Windows Phone.
 
 http://xamarin.com/prebuilt/employeedirectory
 
+This sample has been removed for iOS because the "UISearchDisplayController" API was depercated in IOS 7+ and replaced with "UISearchController".
```

Comment:
https://developer.apple.com/documentation/uikit/uisearchdisplaycontroller?language=objc says `iOS 3.0–8.0` though. So probably change that to iOS 8+.